editor.planx.uk/src/@planx/components/Question/Public/Question.tsx

```diff
@@ -100,68 +103,70 @@ const Question: React.FC<IQuestion> = (props) => {
           >
             {props.text}
           </FormLabel>
-          <RadioGroup
-            aria-labelledby={`radio-buttons-group-label-${props.id}`}
-            name={`radio-buttons-group-${props.id}`}
-            value={formik.values.selected.id}
-          >
-            <Grid
-              container
-              spacing={layout === QuestionLayout.Basic ? 0 : 2}
-              alignItems="stretch"
+          <ErrorWrapper id={`${props.id}-error`} error={formik.errors.selected?.id}>
```

Comment:
nit: the `ErrorWrapper` component already prepends `error-message-` to whatever `id` is set here, so no need to explicitly use `-error` suffix as you have